src/solc_0.7/catalyst/AssetAttributesRegistry.sol

```diff
@@ -154,6 +155,7 @@ contract AssetAttributesRegistry is WithMinter, WithUpgrader {
     /// @notice set the migratcion contract address, admin or migration contract only
     /// @param _migrationContract address of the migration contract
     function setMigrationContract(address _migrationContract) external {
+        // TODO use migrationAdminRole ?
```

Comment:
@wighawag  Not sure how you think this should be changed...

Reply:
We might leave as is, actually, we might want to update the migration contract to change itself though

Reply:
@wighawag I had forgotten about this comment. Do
 you think we need to change anything here?

Reply:
I think we can leave as is. this simplify the code and the migration contract is meant to be used only once